src/test/java/io/aiven/kafka/connect/common/output/parquet/ParquetSchemaBuilderTest.java

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright 2021 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.output.parquet;
+
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.kafka.common.record.TimestampType;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.header.ConnectHeaders;
+import org.apache.kafka.connect.header.Headers;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import io.aiven.kafka.connect.common.config.OutputField;
+import io.aiven.kafka.connect.common.config.OutputFieldEncodingType;
+import io.aiven.kafka.connect.common.config.OutputFieldType;
+
+import io.confluent.connect.avro.AvroData;
+import org.apache.avro.Schema.Type;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ParquetSchemaBuilderTest {
+
+    static final int AVRO_CACHE_SIZE = 10;
+
+    @Test
+    void testSchemaForSimpleType() {
+        final var fields =
+                List.of(new OutputField(OutputFieldType.VALUE, OutputFieldEncodingType.NONE));
+
+        final var schemaBuilder = new ParquetSchemaBuilder(fields, new AvroData(AVRO_CACHE_SIZE));
+
+        final var sinkRecord =
+                new SinkRecord(
+                        "some-topic", 1,
+                        Schema.STRING_SCHEMA, "some-key",
+                        Schema.STRING_SCHEMA, "some-value",
+                        100L, 1000L, TimestampType.CREATE_TIME);
+        final var avroSchema = schemaBuilder.buildSchema(sinkRecord);
+
+        assertNotNull(avroSchema);
+        assertEquals(Type.RECORD, avroSchema.getType());
+        assertEquals(Type.STRING, avroSchema.getField(fields.iterator().next().getFieldType().name).schema().getType());
```

Comment:
```suggestion
        assertEquals(Type.STRING, avroSchema.getField(fields.get(0).getFieldType().name).schema().getType());
```
a bit easier to read